src/rrtmgp/AtmosphericStates.jl

```diff
@@ -9,11 +9,13 @@ using DocStringExtensions
 
 using ..Utilities
 using ..MeshOrientations
-using ..PhysicalConstants
 using ..Gases
 using ..GasConcentrations
 using ..ReferenceStates
 
+using CLIMAParameters
+import CLIMAParameters.Planet: molmass_dryair, molmass_water
```

Comment:
```suggestion
using CLIMAParameters.Planet: molmass_dryair, molmass_water
```
We should not be importing here, only `using`.